wrangler-core/src/test/java/io/cdap/wrangler/utils/StructuredToRowTransformerTest.java

```diff
@@ -53,10 +56,20 @@ public void testStructuredRecordToRow() {
       Schema.Field.of("f11", Schema.of(Schema.LogicalType.TIME_MICROS)),
       Schema.Field.of("f12", Schema.of(Schema.LogicalType.TIME_MILLIS)),
       Schema.Field.of("f13", Schema.decimalOf(3, 2)),
-      Schema.Field.of("f14", Schema.of(Schema.LogicalType.DATETIME)));
+      Schema.Field.of("f14", Schema.of(Schema.LogicalType.DATETIME))
```

Comment:
Should also add `Array` type to the record for unit test.

Reply:
Added